src/TMS.Kunina.Homework3.2/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace TMS.Kunina.Homework3._2
+{
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Enter Date:");
+            string str = Console.ReadLine();
+
+            Program program = new Program();
```

Comment:
You can not create an object, but make the method static and just call it:
`whatDayOfWeek(str)`